Sources/WhisperKit/Core/AudioEncoder.swift

```diff
@@ -22,16 +22,14 @@ public class AudioEncoder: AudioEncoding, WhisperMLModel {
         guard let inputDescription = model?.modelDescription.outputDescriptionsByName["encoder_output_embeds"] else { return nil }
         guard inputDescription.type == .multiArray else { return nil }
         guard let shapeConstraint = inputDescription.multiArrayConstraint else { return nil }
-        let shape = shapeConstraint.shape.map { $0.intValue }
-        return shape[1]
+        return shapeConstraint.shape[0].intValue
```

Comment:
shape[0]: Batch size
shape[1]: Sequence length
shape[2]: Embedding dimension

Reply:
@1amageek  I don't think this is right: https://huggingface.co/argmaxinc/whisperkit-coreml/blob/main/openai_whisper-base/AudioEncoder.mlmodelc/metadata.json#L59